jicofo/src/test/kotlin/org/jitsi/jicofo/conference/ConferenceTest.kt

```diff
@@ -320,27 +318,30 @@ class ConferenceTest : ShouldSpec() {
 
             context("Adding sources already signaled") {
                 val sources = remoteParticipant3.sources
-                jingleSession3.processIq(remoteParticipant3.createSourceAdd(sources)).let {
-                    it shouldNotBe null
-                    it.shouldBeInstanceOf<StanzaError>()
+                jingleSession3.processIq(remoteParticipant3.createSourceAdd(sources))
+
+                xmppConnection.requests.last().let { lastRequest ->
+                    lastRequest.type shouldBe IQ.Type.error
```

Comment:
Why does this need last().let {} rather than just last().type shouldBe?  (And below.)